show_gui: false
camera_device: 5

H_min: 140
H_max: 9

S_min: 3
S_max: 255

V_min: 0
V_max: 255

min_contour_area: 5000
min_contour_radius: 40
max_contour_radius: 60
min_area_ratio: 0.4

kalman_dt: 0.2
kalman_accel_noise_mag: 0.5
hungarian_dist_thres: 60.0
hungarian_max_skipped_frames: 10

arm_camera_position_x: 361.0
arm_camera_position_y: 310.0
